ee/modules/110-istio/openapi/config-values.yaml

```diff
@@ -138,6 +138,14 @@ properties:
         description: Turn on or off tracing collection and displaying in kiali.
         default: false
         x-examples: [true]
+      sampling:
```

Comment:
Could you please elaborate more on this option and why tuning this is required?

Reply:
My question was more about the `sampling` term. May be we can provide a link to some documentation that can clarify what sampling is.